Assets/Scripts/PlaneDataFromName.cs

```diff
@@ -974,6 +987,17 @@ public static PlaneAttributes AttributesFromName(string name)
                 planeAttributes.waterTempType = DialVariant.A;
                 break;
 
+            // Couldn't find!
```

Comment:
Usually a reference to where it is in the source is here, this time I couldn't find it